### PR TITLE
Math magicians: components

### PR DESCRIPTION
**Pull request by fernandorpm**
# Math magicians: components

## To add
src/components
- Calculator JS and CSS files
- CalcButton.js

## To update
- App.js

## Requirements
- Delete all the boilerplate from CRA (text, images, styles).
- Create a directory called `components`.
- Inside `components`, create a new `Calculator.js` file.
- In `Calculator.js`, create a React component that matches this design:
- Don't add any extra design.
- Develop just the UI of the calculator; the math logic to make it work will be added in the next project.
- Use class based components.
- Once the Calculator is ready, import it and use it in your main component (App.js).
- Test it in the browser; it should render without problems.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,27 +1,10 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
     <div className="App">
```
Comment (Dipeshtwis) [inline src/App.js:6]: - [x] The `App.js` component should be class-based as per the project's requirements even though functional component is very correct. Please change this functional component to a class component. :+1:
- [x] You also need to import `React, {Component}` when you convert the functional component to class component extended from `React Component`.

File: src/components/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+      <CalcButton character="4" />
+      <CalcButton character="5" />
+      <CalcButton character="6" />
+      <CalcButton character="-" classButton="orangeButton" />
+
+      <CalcButton character="1" />
+      <CalcButton character="2" />
+      <CalcButton character="3" />
+      <CalcButton character="/" classButton="orangeButton" />
```
Comment (Dipeshtwis) [inline src/components/Calculator.js]: - [x] I think you have interchanged the position of `+` and `÷` text. Also, you can use the `÷` symbol for division, I know it's not on the keyboard but you can search for `division text` on google or you can use `HTML character`

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @fernandorpm, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.
:+1: Linters are passing :white_check_mark: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

- [x] You have removed almost all the file which was not required, but still, some of them are left, Please Remove `App.test.js` and `setupTests.js`, as we are not testing anything for this moment.
- [x] I would recommend you to put `.vscode` in the `.gitignore` file as we don't want to put an unnecessary file on GitHub.

**_Also check the comments under the review_**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mricanho — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @fernandorpm  🔥 

You worked hard in all the instructions that the previous Code Reviewer gives you👍🏽 

yougotit

Now you can merge this branch 🚀 

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**